app/views/projects.html

```diff
@@ -28,6 +28,12 @@ <h2 class="project truncate">
                     </div>
                   </div>
                   <div row flex main-axis="end" class="project-delete" ng-if="project.status.phase == 'Active'">
```

Comment:
Probably worth updating the class name to something like `project-actions` since it's not only delete.